daffodil-test/src/test/resources/org/apache/daffodil/section00/general/tunables.tdml

```diff
@@ -693,6 +694,7 @@
           <ex:char>2</ex:char>
           <ex:char>3</ex:char>
           <ex:char>4</ex:char>
+          <ex:char>�</ex:char>
```

Comment:
Nit, but I think a character entity would be better here than this literal unicode character. `&#xFFFD;` along with an XML comment saying this is the unicode replacement character.